lib/oli/inventories/publisher.ex

```diff
@@ -50,4 +64,16 @@ defmodule Oli.Inventories.Publisher do
         false
     end
   end
+
+  defp maybe_validate_support_email(changeset) do
```

Comment:
The regex here accepts an invalid email as valid (it can not include more than one `@` symbol):
`test@@example.com`

Something like `~r/^[^\s@]+@[^\s@]+\.[^\s@]+$/` would do the trick here.

Anyway, since we have several places where we accept emails as user input, it'd be better if we unify the logic behind all these places and have a shared module that takes over email validation.
Maybe as a future tech debt, we could delegate this to an existing library such as [email_checker](https://github.com/maennchen/email_checker)